.drone.yml

```diff
@@ -126,6 +127,11 @@ environment:
   DOCKER_HOST: tcp://docker:2375
 
 steps:
+  - name: clone
+    image: alpine/git
+    commands:
+      - git clone https://github.com/UKHomeOffice/hocs-case-creator.git .
+      - git checkout $${VERSION}
```

Comment:
I wonder if it'd be better here to use the vanilla `clone` step here, and add our own `checkout` step that just checks out the right version?

Ideally we'd move to a point where we'd just deploy the value of `$DRONE_COMMIT_SHA` based off the number at the end of the promote command, instead of an arbitrary tag/commit that is unrelated to the build number of the promote job -- but I think this is a good solution for the interim.

Reply:
`$${VERSION}` is the Git sha now as we promote through the `DRONE_COMMIT_SHA` to environments with this change. This is just a foot throgh the door change for now.